apps/api/env/.env.unit.test

```
@@ -4,7 +4,7 @@ MASTER_WALLET_MNEMONIC="motion isolate mother convince snack twenty tumble boost
 UAKT_TOP_UP_MASTER_WALLET_MNEMONIC="since bread kind field rookie stairs elephant tent horror rice gain tongue collect goose rural garment cover client biology toe ability boat afford mind"
 USDC_TOP_UP_MASTER_WALLET_MNEMONIC="leaf brush weapon puppy depart hockey walnut hospital orphan require unfair hunt ribbon toe cereal eagle hour door awesome dress mouse when phone return"
 NETWORK=sandbox
-RPC_NODE_ENDPOINT=https://rpc.sandbox-01.aksh.pw:443
+RPC_NODE_ENDPOINT=https://rpc.sandbox-02.aksh.pw:443
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The RPC_NODE_ENDPOINT key should go before the UAKT_TOP_UP_MASTER_WALLET_MNEMONIC key

(UnorderedKey)